pybrake/routes.py

```diff
@@ -43,28 +44,33 @@ def __init__(self, project_id, project_key, host):
   def _init(self):
     if self._stats is None:
       self._stats = {}
-      self._thread = Timer(self._flush_period, self._flush)
+      self._thread = Timer(self._flush_period, self.flush)
       self._thread.start()
 
-  def inc_request(self, method='', route='', status_code=0, time=None, ms=0):
-    self._init()
-
-    statKey = route_stat_key(method, route, status_code, time)
-    with self._lock:
-      if statKey in self._stats:
-        stat = self._stats.get(statKey)
-        stat.add(ms)
-      else:
-        self._stats[statKey] = RouteStat(method, route, status_code, time, ms)
-
-  def _flush(self):
+  def flush(self):
     stats = dict()
     with self._lock:
       stats = self._stats
       self._stats = None
+      self._thread.cancel()
```

Comment:
Why this is needed?

Reply:
It is need in case execution flush function manually instead timer's function execution.